Makefile.include

```diff
@@ -1,3 +1,6 @@
+# vim: ts=8:sw=8:ft=make:noai:noet
```

Comment:
Let me know if there is any reason to remove the modeline, it makes life easier for me with my vim settings